src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,69 @@
+package mate.academy.model;
+
+import java.util.Arrays;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
```

Comment:
you can  add here  for email field @Column(unique = true). "it'll ensure that the values in the email column must be unique for each row in the table. It prevents duplicate values from being stored in the email column, ensuring that each user has a unique email address in the database."